src/rules/avoid-dispatching-multiple-actions-sequentially.ts

```diff
@@ -34,41 +35,43 @@ export default ESLintUtils.RuleCreator(docsUrl)<Options, MessageIds>({
   create: (context) => {
     let storeName = ''
     return {
-      [injectedStore](node: TSESTree.Identifier) {
-        storeName = node.name
+      [injectedStore]({ name }: TSESTree.Identifier) {
+        storeName = name
       },
       [`ClassDeclaration > ClassBody > MethodDefinition > FunctionExpression > BlockStatement`](
         node: TSESTree.BlockStatement,
       ) {
         if (!storeName) return
 
-        const dispatchExpressions = node.body
-          .map((expression: TSESTree.ExpressionStatement) => {
-            if (
-              isCallExpression(expression.expression) &&
-              isMemberExpression(expression.expression.callee) &&
-              isIdentifier(expression.expression.callee.property) &&
-              expression.expression.callee.property.name === 'dispatch' &&
-              isMemberExpression(expression.expression.callee.object) &&
-              isIdentifier(expression.expression.callee.object.property) &&
-              expression.expression.callee.object.property.name === storeName
-            ) {
-              return expression.expression
-            } else {
-              return null
-            }
-          })
-          .filter(Boolean)
+        const dispatchExpressions = (node.body.filter((expression) =>
+          isStoreDispatchExpression(storeName, expression),
+        ) as TSESTree.ExpressionStatement[]).map(({ expression }) => expression)
 
         if (dispatchExpressions.length > 1) {
-          dispatchExpressions.forEach((expression) => {
+          dispatchExpressions.forEach((expression) =>
             context.report({
               node: expression,
               messageId: 'AvoidDispatchingMultipleActionsSequentially',
-            })
-          })
+            }),
+          )
         }
       },
     }
   },
 })
+
+function isStoreDispatchExpression(
```

Comment:
It was giving some compilation errors when I enabled strict mode, so I correct them and move it to a function in order to make it more readable.